aim/sdk/sequences/geometry_sequence.py

```diff
@@ -0,0 +1,17 @@
+from typing import Union, Tuple
+
+from aim.sdk.sequence import MediaSequenceBase
+from aim.sdk.objects.geometry import Geometry
+
+
+class Geometries(MediaSequenceBase):
+    """Class representing series of Geometry objects or Geometry lists."""
+
+    @classmethod
+    def allowed_dtypes(cls) -> Union[str, Tuple[str, ...]]:
+        geometry_typename = Geometry.get_typename()
+        return geometry_typename, f'list({geometry_typename})'
```

Comment:
Lets allow tracking of single geometry objects to improve UX when tracking and reading sequences.